wolfcrypt/src/port/st/stsafe.c

```diff
@@ -87,6 +90,10 @@ int SSL_STSAFE_CreateKeyCb(WOLFSSL* ssl, ecc_key* key, word32 keySz,
     /* generate new ephemeral key on device */
     err = stsafe_interface_create_key(&slot, curve_id, (uint8_t*)&pubKeyRaw[0]);
     if (err != 0) {
```

Comment:
Return isn't STSAFE_A_OK?

Reply:
You are correct! I will change to `STSAFE_A_OK`. Thank you